### DISCUSSION TITLE
Initial API token for fully automated deployments

### DISCUSSION THREAD

**Discussion by muellerst-hg** [DependencyTrack/dependency-track#2102] [Q&A] [answered]
After a fresh installation, is there a way to set or fetch an initial admin API token without ínteractive login in the frontend?

My use case is: Automated deployments of dtrack apiserver with oidc user management. I'd like to setup oidc through API in a pipeline, but would need a token with ACCESS_MANAGEMENT.

e.g. by providing an ENV variable:
```
docker run -it -d --network dependency_trackt -e ADMINISTRATORS_API_TOKEN=not_the_token dependencytrack/apiserver:latest
```

**Comment by syalioune** [comment, marked as answer]
Hello @muellerst-hg, you can check https://github.com/DependencyTrack/dependency-track/issues/831#issue-741441433
